ingestion/src/main/java/feast/ingestion/transform/metrics/WriteFeatureValueMetricsDoFn.java

```diff
@@ -90,9 +90,11 @@ static Builder newBuilder() {
   public static String GAUGE_NAME_FEATURE_VALUE_MIN = "feature_value_min";
   public static String GAUGE_NAME_FEATURE_VALUE_MAX = "feature_value_max";
   public static String GAUGE_NAME_FEATURE_VALUE_MEAN = "feature_value_mean";
+  public static String GAUGE_NAME_FEATURE_VALUE_PERCENTILE_25 = "feature_value_percentile_25";
```

Comment:
I think 100 would be more useful than 25 tbh

Reply:
We currently have `feature_value_max` metric to record the highest feature value in the corresponding window. I can change the name to  `feature_value_percentile_100` if you think it makes it more consistent.

Reply:
Oops, didnt see that. My bad!